src/Avalonia.Controls/Platform/ManagedDispatcherImpl.cs

```diff
@@ -40,7 +41,7 @@ public void Signal()
 
     public event Action? Signaled;
     public event Action? Timer;
-    public long Now => _clock.ElapsedMilliseconds;
+    public long Now => (long)GetElapsedTime().TotalMilliseconds;
```

Comment:
`ManagedDispatcherImpl` can be a tight loop, can we avoid the extra `long` to `double` to `long` conversion for each access to `Now`, which is likely measurable on low end embedded devices?

Related: #11674 (which was closed for inactivity but could probably be reopened)

Reply:
TIL that Avalonia dispatcher kinda ignores any precision lower than milliseconds. Timer of 0.5ms is the same as timer of 1tick - and both run instantly, cause they have dueTime of 0ms.

Reply:
Done

Reply:
@MrJul upd: this `long` to `double` to `long` can't be removed completely.
I.e. the less overhead I came up with is:
```
    private static readonly double s_conversionFactor = (double)TimeSpan.TicksPerSecond / (Stopwatch.Frequency * TimeSpan.TicksPerMillisecond);

    public double GetElapsedMilliseconds(long startingTimestamp, long endingTimestamp) => (endingTimestamp - startingTimestamp) * s_conversionFactor;
```

Where `s_conversionFactor` can be pre-computed once, and it very likely will be a decimal number around 0.001.

But we can avoid TimeSpan creation at the very least. Sure.

Reply:
But at the same time, we still end up creating TimeSpan for MediaContext and ServerCompositor anyway, as that's the type we use in our animations.